### PR TITLE
Add introduction video of the standard to README

### LINKED ISSUE #360: Add CC Global Summit video on the Standard to README
And check if there's anywhere else it makes sense to [link to the video](https://www.youtube.com/watch?v=QWt6vB-cipE&list=PL_5ziu2gADmDcp_ER8x2rcT1HOBaIe5cl&index=1).

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #360 

-----
[View rendered README.md](https://github.com/publiccodenet/standard/blob/video-to-readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ This standard sets a quality level for codebases that meets the needs of public
 
 The standard lives at [standard.publiccode.net](https://standard.publiccode.net/), and for an overview of all content see [`index.md`](index.md).
 
+[![The printed Standard for Public Code lying on a table between two hands](https://img.youtube.com/vi/QWt6vB-cipE/mqdefault.jpg)](https://www.youtube.com/watch?v=QWt6vB-cipE)
```
Comment (bvhme) [inline README.md:13]: This image has a role, perhaps that should be reflected in the alt attribute.

```suggestion
[![Thumbnail for the video on the Standard for Public Code: a printed version lying on a table between two hands](https://img.youtube.com/vi/QWt6vB-cipE/mqdefault.jpg)](https://www.youtube.com/watch?v=QWt6vB-cipE)
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ This standard sets a quality level for codebases that meets the needs of public
...
+
+The image above is linked to an introduction video of the Standard for Public Code on YouTube.
```
Comment (bvhme) [inline README.md:15]: This is really hard to use for those using non-gui-mouse based browsers. It also adds the cognitive overhead of having to relate to other elements before being able to act. A simple link with a good description will serve it's purpose better here.

```suggestion
[The video: Standard for Public Code - Creative Commons Global Summit 2020 (4:12) on YouTube](https://www.youtube.com/watch?v=QWt6vB-cipE)
```

**Review by bvhme — CHANGES_REQUESTED** [review]
Great addition, some suggestions

File: README.md (hunk 1 above)
Comment (Ainali) [reply README.md:13]: I thought alt-text only should describe what you actually see, not adding semantic meaning of the image to it.

Comment (bvhme) [reply README.md:13]: The alt text is what displays as an alternative to an image. For people who can see the image it is clearly a 'thumbnail' and they will infer meaning from it. As such it being a thumbnail is an integral part of it's description.

Imagine how someone would describe the elements on the page to you. I'd say 'there is a thumbnail of the video with the standard on it and the title and link underneath."

**Review by bvhme — APPROVED** [review]

**Pull request merged**